### ISSUE TITLE
Running the generateDocs.py script throws an error.

### ISSUE THREAD

**Issue by enigmashot** [cinder/Cinder#1729]
I just cloned the repository and successfully built it with VS2015. Sample projects are running fine.  And I ran the doxygen wizard successfully and created the xml files. At this point, I tried to run the generateDocs.py script and got the following error:

```
D:\Sources\Cinder\docs>python generateDocs.py
Traceback (most recent call last):
  File "generateDocs.py", line 3740, in <module>
    load_meta()
  File "generateDocs.py", line 3672, in load_meta
    file_meta["doxy_version"] = meta_file.attrib.get("version")
AttributeError: 'NoneType' object has no attribute 'attrib'
```

**Comment by andrewfb** [comment]
Thanks for this - what version of Doxygen are you using? It appears that the latest version of Doxygen itself crashes when generating the Cinder docs, though it behaves correctly on the Mac.

**Comment by gregkepler** [comment]
I don’t have access to a PC right now. Can you post me a copy of _cinder_8h.xml that doxygen generates? It sounds like there’s no “version” tag being generated by the new version of doxygen.

**Comment by enigmashot** [comment]
@andrewfb: I was using the latest version (1.8.13) when I encountered this problem. So I tried version 1.8.10 as mentioned in the guide and it worked fine! 

@gregkepler: I checked out two fresh copies of the latest version of the master branch and tried it with both 1.8.10 and 1.8.13 to replicate the error and provide the file that you asked for, but they both worked this time.

After a few more tests, the script is working fine, even with the latest version of Doxygen. I was not able to replicate the error. So I think it's best I close this issue. 

Thank you for your responses. I just started using cinder again after 4 years and I'm amazed at how much the framework matured and how well the community grew. Just wanna say I appreciate it!
